src/java/org/apache/cassandra/net/InboundConnectionSettings.java

```diff
@@ -83,8 +83,8 @@ public boolean authenticate(InetAddress address, int port)
 
     public String toString()
     {
-        return format("address: (%s), nic: %s, encryption: %s",
-                      bindAddress, FBUtilities.getNetworkInterface(bindAddress.address), SocketFactory.encryptionLogStatement(null, encryption));
+        return format("address: (%s), nic: %s, %s",
```

Comment:
Could we add back `encryption: ` label?

Reply:
Can do - I thought it was redundant with the descriptions 'unencrypted', 'optionally encrypted' or 'encrypted' 

would go from
```
Listening on address: (/127.0.0.1:7012), nic: lo0, unencrypted
Listening on address: (/127.0.0.1:7012), nic: lo0, optionally encrypted(jdk)
Listening on address: (/127.0.0.2:7012), nic: lo0, encrypted(jdk)
```
to
```
Listening on address: (/127.0.0.1:7012), nic: lo0, encryption: unencrypted
Listening on address: (/127.0.0.1:7012), nic: lo0, encryption: optionally encrypted(jdk)
Listening on address: (/127.0.0.2:7012), nic: lo0, encryption: encrypted(jdk)
```

Still prefer it back?

Reply:
From a parsing standpoint, I prefer the second. It's easier to parse the log entries, for example, in Splunk. It is also consistent with what we print in `OutboundConnectionSettings` https://github.com/apache/cassandra/pull/763/files#diff-77d6e44359d621fcf54fef9e65dd0a10L173

Reply:
Ok, will keep the original `encryption:` tag.